db/model/generator.js

```diff
@@ -79,7 +79,7 @@ module.exports = function generator(seq, dataTypes) {
       type: dataTypes.ARRAY(dataTypes.STRING(constants.fieldlen.normalName)),
       allowNull: false,
     },
-    keywords: {
+    tags: {
```

Comment:
There aren't any Generators or GeneratorTemplates in prod yet.
Does it need a migration if there's nothing there already?